scripts/bash/_version_file.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
One more questions/suggestion. We have `_print_help.sh`. Wouldn't it be more consistent to have `_print_version.sh`?